src/getting_started/changelogs/Changelogs.js

```diff
@@ -15,6 +15,81 @@ export default function Authentication() {
         </p>
       </section>
       <section>
+        <h2>2018-04-02</h2>
+        <hr /><br />
+        <b>Breaking:</b><br />
+        <ul>
+          <li> Unify IPv4, IPv6 GET/POST use "type" "public"</li>
+          <li> Moved /managed/linode_settings to /managed/linode-settings
+            <ul>
+              <li> /managed/linode_settings/:id moved to /managed/linode-settings/:id</li>
+              <li> This was to keep our convention of using dashes in URLs
+              instead of underscores</li>
+            </ul>
+          </li>
+          <li> Password no longer accepted in POST /account/users
+            <ul>
+              <li> You may no longer provide a password when creating a new user</li>
+              <li> New users will immediately receive password reset email to
+              set their password</li>
+            </ul>
+          </li>
+          <li> Removed "addresses" from GET /linode/instances/:id/ips response
+            <ul>
+              <li> These addresses are now returned in GET /networking/ips</li>
+            </ul>
+          </li>
+          <li> Moved GET/PUT for range/pool v6 addresses to /networking/ips
+            <ul>
+              <li> GET /networking/ipv6/:address move to GET /networking/ips/:address</li>
+              <li> PUT /networking/ipv6/:address move to GET /networking/ips/:address</li>
+            </ul>
+          </li>
+          <li> /linode/instances/$id/rebuild returns a Linode</li>
+          <li> Fixed inconsistent responses for action endpoints
+            <ul>
+              <li> POST /linode/instances/:id/backups-enable now returns {} on success</li>
+              <li> POST /linode/instances/:id/backups-disable now returns {} on success</li>
+              <li> POST /networking/ip-assign now returns {} on success</li>
```

Comment:
ip-assign has moved to networking/ipv4/assign according to changes below. So, this should be removed here, and a sub-list bullet added to the section below, stating that it now returns {} on success.